### ISSUE TITLE
Script is not working  with attached HTML (Chart)

### ISSUE THREAD

**Issue by VikasSahu** [christopherdro/react-native-html-to-pdf#239]
Hello 

I want to have a line chart  and bar chart in my generated PDF but somehow script is not working with attached HTML.

I am using below chart script in my HTML code (only script part is here, its  Incomplete code)

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js"></script>
<body>
<canvas id="myChart" style="width:100%;max-width:600px"></canvas>
<script>var xValues=["Italy","France","Spain","USA","Argentina"];var yValues=[55,49,44,24,15];var barColors=["red","green","blue","orange","brown"];new Chart("myChart",{type:"bar",data:{labels:xValues,datasets:[{backgroundColor:barColors,data:yValues}]},options:{legend:{display:false},title:{display:true,text:"World Wine Production 2018"}}});</script>
```

Could please help, how to achieve that?

**Comment by zhani-tegeria** [comment]
Any solutions?

**Comment by abhijit-jena21** [comment]
@VikasSahu Did you find any solution or workaround for this?

**Comment by OmSingh5092** [comment]
Any Solutions?

**Comment by suresh-borad** [comment]
I found the solution to enable js in web view.

**Comment by suresh-borad** [comment]
After this line you need to add : https://github.com/christopherdro/react-native-html-to-pdf/blob/master/android/src/main/java/android/print/PdfConverter.java#L114

```
settings.setJavaScriptEnabled(true);
settings.setJavaScriptCanOpenWindowsAutomatically(true);
```
To enabled js in web view.

**Comment by OmSingh5092** [comment]
Hey @suresh-borad the script is functional but due to some reason the Chart is not visible in the pdf. Could this be because of the use of <canvas/>.

**Comment by suresh-borad** [comment]
> Hey @suresh-borad the script is functional but due to some reason the Chart is not visible in the pdf. Could this be because of the use of.

Yes, @OmSingh5092 I had the same issue with android and I'm using [Google Chart](https://www.gstatic.com/charts/loader.js) lib in HTML. After these changes, it's working fine.

The iOs was working fine without any changes but the issue with android.

**Comment by jpn918** [comment]
html = '<h1 id="header1">hello </h1> <script>document.getelementbyid("header1").innertText ="hellofriend"; </script>'
Does this <script> tag will work from this react-native-html-to-pdf library?

**Comment by rkchoudhury** [comment]
Hey @suresh-borad,

Is the changes-https://github.com/christopherdro/react-native-html-to-pdf/issues/239#issuecomment-1058125888 merged to the main branch?

Because this changes is not yet available in 0.12.0 version.

**Comment by suresh-borad** [comment]
@rkchoudhury the pr was auto closed by gitbot. Its not merged
